portfolio/src/main/webapp/script.js

```diff
@@ -83,12 +77,10 @@ function getServerStats() {
 
     const statsListElement = document.getElementById('data-container');
     statsListElement.innerHTML = '';
-    statsListElement.appendChild(
-        createListElement('Message 1: ' + quotes[1]));
-    statsListElement.appendChild(
-        createListElement('Message 2: ' + quotes[2]));
-    statsListElement.appendChild(
-        createListElement('Message 3: ' + quotes[3]));
+    var i;
+    for (i=0; i<quotes.length; i++) {
```

Comment:
Add space between the characters, and please do this from now on.

eg. for (i = 0; i < quotes.length; ...)